Disco_Tie/code.py

```diff
@@ -221,12 +232,12 @@ def change_speed(mod, old_speed):
                     elif packet.button == ButtonPacket.BUTTON_4:
                         mode = 3
 
-    # Determine animation based on mode
-    if mode == 0:
-        peak = audio_meter(peak)
-    elif mode == 1:
-        rainbow_cycle(0.001)
-    elif mode == 2:
-        larsen(wait)
-    elif mode == 3:
-        solid(user_color)
+        # Determine animation based on mode
```

Comment:
Ok - I got it working while disconnected by moving this block into an `animate(wait, peak)` function. 
Then calling `peak = animate(wait, peak)` at:

- last line of `while ble.connected:`
- before `pass` under `while not ble.connected:`

Is this a good solution?

Also - is there a better way for me to suggest a specific change without doing a PR on this branch? That seemed excessive.

Reply:
let's merge this and I'll do anther PR for the changes I suggested.